### PR TITLE
Change the IRC server to Libera.Chat

### PR DESCRIPTION
**Pull request by mhajder**
I changed all references from Freenode to Libera.Chat due to the main channel being moved to the Libera.Chat network.
I haven't changed anything in the `tests` folder.

### CODE REVIEW

**Review by MaxLeiter — APPROVED** [review]

**Comment by richrd** [comment]
Some tests didn't pass, they need to be fixed too. For example `AssertionError: expected 'irc.libera.chat' to equal 'chat.freenode.net'`

**Comment by mhajder** [comment]
I would like to ask you to check if this is what you meant. I didn't quite understand what I was supposed to do, so I changed the default network name and some channels to a more generic.

You could still change this file. Unfortunately, I have no idea what name to change to.
https://github.com/thelounge/thelounge/blob/master/test/tests/textLogFolder.js#L6-L42

File: test/client/js/helpers/ircmessageparser/findLinks.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ const {
 
 describe("findLinks", () => {
 	it("should find url", () => {
-		const input = "irc://freenode.net/thelounge";
+		const input = "irc://irc.example.com/thelounge";
 		const expected = [
 			{
 				start: 0,
 				end: 28,
```
Comment (xPaw) [inline test/client/js/helpers/ircmessageparser/findLinks.js:15]: This should be 31 now (length of the link).

**Comment by xPaw** [comment]
Changes look correct to me (sans one mistake). The textLogFolder file is fine to keep as is, all it is doing is testing folder name generation.

File: test/client/js/helpers/ircmessageparser/findLinks.js (hunk 1 above)
Comment (mhajder) [reply test/client/js/helpers/ircmessageparser/findLinks.js]: Oh, I forgot to commit this. Already fixed.

**Review by xPaw — APPROVED** [review]

**Pull request merged**